### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by JonasSchiessl**
Added the forgot screen with a pop up. Navigation needs to be added.

### CODE REVIEW

File: navigation/screens/forgot-password.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        {/* Submit button */}
+        <View style={styles.fixToText}>
+          <Button title="Submit" color={"#000000"} onPress={onSubmitPress} />
+        </View>
+      </SafeAreaView>
+    </View>
+  );
+}
+
+export default TabSubmitScreen;
```
Comment (k4yluh) [inline navigation/screens/forgot-password.tsx:62]: Could also just do export default in front of the function def

File: yarn.lock
Comment (k4yluh) [inline yarn.lock:1]: we are using node.js for front not yarn? may need to delete this file

File: navigation/screens/forgot-password.tsx (hunk 1 above)
Comment (k4yluh) [inline navigation/screens/forgot-password.tsx:62]: Could also just do export default in front of the function def

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-native": "0.73.4",
+    "react-native-modal": "^13.0.1",
```
Comment (k4yluh) [inline package.json:22]: what is this dependency for?

**Review by k4yluh — CHANGES_REQUESTED** [review]
Everything basically looks good to go; just left some small comments. Only major one is on the yarn file

File: package.json (hunk 2 above)
Comment (JonasSchiessl) [reply package.json:22]: This dependency is used for creating pop ups in react native, specifically I used it to let the user know they successfully sent the email.

File: navigation/screens/forgot-password.tsx (hunk 1 above)
Comment (JonasSchiessl) [reply navigation/screens/forgot-password.tsx]: changed that

File: yarn.lock
Comment (JonasSchiessl) [reply yarn.lock:1]: That was auto generated, not sue from what to be honest but i deleted it

Comment (k4yluh) [reply yarn.lock:1]: oops! I guess Megan set it up; so please reput it. Sorry for the confusion

**Review by k4yluh — CHANGES_REQUESTED** [review]
Just put the yarn file back and its good :)

**Review by k4yluh — APPROVED** [review]
Thank you for putting it back! Sorry about the confusion.

**Pull request merged**